test/visualTest.ts

```diff
@@ -402,14 +605,15 @@ describe("Sunburst", () => {
                     expect(elements.length).toBeTruthy();
 
                     done();
-                });
+                }, 2, DefaultWaitForRender);
         });
     });
 
     describe("Capabilities tests", () => {
         it("all items having displayName should have displayNameKey property", () => {
-            let r = fetch("base/capabilities.json");
-            let jsonData = JSON.stringify(r);
+            jasmine.getJSONFixtures().fixturesPath = "base";
```

Comment:
please revert these changes. Fetch is the default way to load capabilities file

Reply:
Done: [fafcb2a](https://github.com/microsoft/powerbi-visuals-sunburst/pull/95/commits/fafcb2aaf1efb4e1ce60b93d91592fcaa75ec29d)